Sources/WalletConnectSigner/Signer/MessageSigner.swift

```diff
@@ -27,22 +31,28 @@ struct MessageSigner: AuthMessageSigner {
     private let signer: EthereumSigner
     private let eip191Verifier: EIP191Verifier
     private let eip1271Verifier: EIP1271Verifier
-    private let messageFormatter: SIWEMessageFormatting
+    private let messageFormatter: SIWECacaoFormatting
```

Comment:
it was called SIWEMessageFormatting because siwe message does not know anything about cacao, not sure if it's better name